### ISSUE TITLE
RECAPTCHA_TESTING should be a django settings

### ISSUE THREAD

**Issue by emmanuel-santos** [praekelt/django-recaptcha#153]
I think that is better define this variable in a django settings, if you use differente settings for production, develop, test is a bit more trasnparent declare in django setttings..

**Comment by adamchainz** [comment]
Yes and tests can change it with `@override_settings` rather than adding custom logic in `setUp()`/`tearDown()`

**Comment by Vanderhoof** [comment]
Guys, I'm sorry, but how exactly have you addressed this issue? I can't see any unittest dedicated setting available, but the environment variable is gone. So what is the recommended approach to mock recaptcha field during tests?

**Comment by plumber-craic** [comment]
Also wondering how we are meant to test a v3 recaptcha form. A test mode setting would be nice, and if this is present but not documented, a hint in the docs would be nice (they mention only that v2 supports test keys).
